### PR TITLE
Correction about:

### PR DESCRIPTION
**Pull request by falehenrique**
- Hook Payment Return - Checkout Standard Brasil
- Hook Payment Return - Checkout Custom Brasil
- Added label field with notification_url in administration page
- Payment were being cancelled when the user used the same external reference
- Added the option "choice" in the installments data field

### CODE REVIEW

File: mercadopagobr.php [hunk 1]
```diff
@@ -826,6 +814,38 @@ public function listenIPN($checkout, $topic, $id)
...
+	*/
+	public static function getOrderStateApproved($id_order)
+	{
+		$id_order_state = Db::getInstance()->getValue('
+		SELECT `id_order_state`
+		FROM '._DB_PREFIX_.'order_history
+		WHERE `id_order` = '.(int)$id_order.' 
+		AND `id_order_state` = '.Configuration::get('MERCADOPAGO_STATUS_1'));
+
+		if (!$id_order_state){
```
Comment (Quetzacoalt91) [inline mercadopagobr.php]: Keep it simple, you can write

```php
return (bool)$id_order_state;
```

**Comment by falehenrique** [comment]
Ok, I changed it and commited.

Tks and reguards.

File: mercadopagobr.php (hunk 1 above)
Comment (falehenrique) [inline mercadopagobr.php]: OK, i did it! What the date it will be published?

**Comment by Quetzacoalt91** [comment]
Hi @falehenrique,

Thanks, your PR has just been merged and the changes should be available from tomorrow.

Best regards

**Pull request merged**